cmd/kubeadm/app/apis/kubeadm/v1beta4/types.go

```diff
@@ -443,3 +443,38 @@ type Patches struct {
 	// +optional
 	Directory string `json:"directory,omitempty"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ResetConfiguration contains a list of elements that is specific "kubeadm reset"-only runtime information.
+type ResetConfiguration struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// SkipPhases is a list of phases to skip during command execution.
+	// The list of phases can be obtained with the "kubeadm reset phase --help" command.
+	// The flag "--skip-phases" takes precedence over this field.
+	// +optional
+	SkipPhases []string `json:"skipPhases,omitempty"`
+
+	// ForceReset flag instructs kubeadm to reset the node without prompting for confirmation
+	// The flag "--force" takes precedence over this field.
+	// +optional
+	ForceReset bool `json:"forceReset,omitempty"`
```

Comment:
A naming problem. Why not `ResetConfiguration.Force`?

Reply:
leaning towards Force for a better mapping with CLI flags.

BTW
> // The flag "--force" takes precedence over this field.

please remove similar comments, because kubeadm should guarantee this for **all** it's flags vs config options.

Reply:
Done